static/js/controllers/configuration-upgrade.js

```diff
@@ -0,0 +1,110 @@
+var PouchDB = require('pouchdb-browser');
+
+// REVIEWER: are we OK hard-coding these two concepts? If not, where should
+// we put this? We don't have access to env vars in medic-webapp.
+var IS_PROD_URL = /^https:\/\/[^.]+.app.medicmobile.org\//,
+    BUILDS_DB = 'https://staging.dev.medicmobile.org/_couch/builds';
```

Comment:
I think this would be part of moving it out into horti, which we're going to leave for another time.